src/components/Executions/ExecutionDetails/contexts.ts

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 
-import { Execution, NodeExecution } from 'models';
+import { Execution, NodeExecution, RequestConfig } from 'models';
+import { ExecutionFiltersState } from '../filters/useExecutionFiltersState';
```

Comment:
I don't see `ExecutionFiltersState` being used anywhere in this file, is that intentional?